_posts/2019-12-05-Rust-für-Java-Entwickler-Teil-1.md

```diff
@@ -24,45 +23,42 @@ Einen guten Einstieg hierfür bietet das Buch [The Rust Programming Language](ht
 
 # Ein neues Projekt aufsetzen
 
-Zuerst müssen wir ein neues Projekt erstellen.
-Das machen wir am besten mit dem Package-Manager Cargo.
-Dazu gehen wir in unser workspace-Verzeichnis und führen den Befehl ```cargo new <projekt-name> --bin``` aus.
-Bevor wir beginnen können, müssen wir noch die entsprechende Actix-Dependency in der Cargo.toml angeben
-(zum Zeitpunkt der Erstellung dieses Artikels hatte Actix-Web die Version 1.0.8).
+Zuerst erstellen wir uns ein neues Projekt.
+Das machen wir am besten mit dem Package-Manager *Cargo*.
+Dazu führen wir den Befehl ```cargo new <projekt-name> --bin``` in unserem Workspace aus.
+Bevor wir beginnen können, müssen wir noch die entsprechende Actix-Dependency in der Cargo.toml angeben.
 
 ```rust
 [dependencies]
-actix-web = "1.0.8"
+actix-web = "1.0.8" // latest version
 ```
 
 # Unsere REST-API
 
 In unserer Beispielanwendung wollen wir Personen verwalten.
-Dazu werden wir eine sehr kleine und einfache REST-API zur Verfügung stellen.
+Dazu werden wir eine kleine und einfache REST-API zur Verfügung stellen.
 Wir wollen die Möglichkeit haben, eine Person zu erstellen, zu löschen und zu modifizieren.
 Außerdem wollen wir Endpoints zur Verfügung stellen, die uns alle oder auch einzelne (durch ihre ID identifizierbare) Personen liefern.
 Wir werden also fünf verschiedene Endpoints implementieren müssen, die wie folgt aussehen:
 
-<pre>
-* GET     /persons
-* GET     /persons/id
-* POST    /persons
-* PUT     /persons/id
-* DELETE  /persons/id
-</pre>
+- GET     /persons
+- GET     /persons/id
+- POST    /persons
+- PUT     /persons/id
+- DELETE  /persons/id
```

Comment:
Die Auflistung funktioniert nun, allerdings wird diese immernoch nicht wirklich schon dargestellt.
Besonders die Einrückungen sind nun bisschen unschön.

Leider habe ich aber auch keinen anderen Weg zur Formattierung gefunden.. In meinem damaligen Blog-Post habe ich die Auflistung mittels Sterne umgesetzt. Allerdings wird hier ebenfalls nicht auf die Einrückung geachtet.

Ansonsten mal überlegen hier eine Tabelle raus zu machen?